com.korbsstudio.penpot-desktop.yml

```diff
@@ -0,0 +1,34 @@
+app-id: com.korbsstudio.penpot-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - name: penpot-desktop
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://updates.korbsstudio.com/penpot-desktop/Penpot%20Desktop-0.1.1.AppImage
```

Comment:
so you use an x86_64 build to also build for aarch64....

any chance to build from source instead?

Reply:
Since I never had any success with using electron-builder in flatpak manifests I chose to repack the appimage instead of attempting to build it from source